src/components/List/_list.scss

```diff
@@ -1 +1,35 @@
-@import '~carbon-components/scss/components/list/list';
+@import '../../globals/vars';
+
+.#{$iot-prefix}--list {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  max-height: 100%;
+  &--header {
+    flex: 0;
+  }
+  &--content {
+    flex: 1 1;
+    background: $ui-01;
+    overflow-y: auto;
+  }
+
+  &__full-height {
+    height: 100%;
+  }
+
+  &--page {
+    position: relative;
+    div[class^='SimplePagination__StyledContainer'] {
```

Comment:
do we need to qualify these selectors with elements?

Reply:
I think it's already qualified, right?  div w/ class attribute, inside of the `.list--page` should get that style applied.